api/authCustom/models.py

```diff
@@ -0,0 +1,34 @@
+import binascii
+import os
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+
+
+class Token_Custom(models.Model):
+    """
+    The default authorization token model.
+    """
+    key = models.CharField(_("Key"), max_length=40)
```

Comment:
Wouldn't it be better to call it `auth_key`? Keeping the terms fixed helps throughout the project.

Reply:
Is it possible to use the `default` field here to generate the `auth_key`?